README.md

```diff
@@ -234,6 +235,15 @@ One of exponential/constant backoff policies can be provided for the search stor
 
 </details>
 
+<details>
+  <summary>Transformer</summary>
+
+| Environment Variable            | Default | Required | Description                                                          |
+| ------------------------------- | ------- | -------- | -------------------------------------------------------------------- |
+| PGSTREAM_TRANSFORMER_RULES_FILE | N/A     | No       | Filepath pointing to the yaml file containing the transformer rules. |
```

Comment:
I wonder if eventually we will want all settings in a single file, but this we can figure out later